modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -422,7 +431,31 @@ sf project deploy start --source-dir path/to/permissionset/ -o orgAlias
 .. In *Setup*, enter `permission sets` in the *Quick Find* box and click *Permission Sets*.
 .. Locate *AEH_Anypoint* in the list.
 
-== See Also
+== Assign API Experience Hub Users Approver Permission
+Assign the API Experience Hub User Approver permission to enable a user to approve or reject pending registration requests and enable or disable member users only. This permission doesn't allow this user to perform any other operations for the portal. 
```

Comment:
```suggestion
Approve or reject pending registration requests and enable or disable member users with the API Experience Hub User Approver permission.
```
I suggest a rewording to describe the action first. I also removed the last sentence because Is it necessary to clarify what this permission does NOT allow? Please apply this suggestion only if you agree and think it is correct.